consensus/src/member.rs

```diff
@@ -460,7 +460,7 @@ where
             &self.not_resolved_parents,
             &self.not_resolved_coords,
         );
-        info!(target: "AlephBFT-member", "{}", status);
+        debug!(target: "AlephBFT-member", "{}", status);
```

Comment:
This is absolutely crucial log, we cannot turn it off :) 
Note that it is NOT logged on start or end. This is a status log. 

I will leave a few more comments saying "status log", which means they should go back to `INFO`.